.ruby-version

```diff
@@ -1 +1 @@
-2.4.0
+2.3.3
```

Comment:
@MatiasComercio yes, but it's not working. Today I tried deploying my app and I couldn't because I had that version.